spec/controllers/reservations_controller_spec.rb

```diff
@@ -1130,6 +1130,8 @@
         @params[:reservation_id] = @reservation.id
         create(:relay, instrument: @instrument)
         @random_user = create(:user)
+        @instrument.update_attributes(min_reserve_mins: 30)
```

Comment:
Can we make this a `update!` so it alerts us loudly if the setup doesn’t succeed?

Reply:
Good call.